sooyoung/controllers/userController.js

```diff
@@ -17,6 +17,23 @@ const signUp = async (req, res) => {
   }
 };
 
+const signIn = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+
+    if (!email || !password) {
+      return res.status(400).json({ message: 'KEY_ERROR' });
+    }
+
+    const userToken = await userService.signIn(email, password);
+    return res.status(200).json({ accessToken: userToken });
```

Comment:
```suggestion
    const accessToken = await userService.signIn(email, password);
    return res.status(200).json({ accessToken });
```
변수 이름과 object의 property name이 일치하면 위와같이 간단하게 작성 할 수 있습니다! 
기능 동작과는 관련 없으나 javascript 에서는 이게 좀 더 깔끔한 코드라고 생각됩니다 :)